rewrite-pattern.js

```diff
@@ -363,21 +462,34 @@ const rewritePattern = (pattern, flags, options) => {
 	validateOptions(options);
 
 	config.flags.unicode = hasFlag(flags, 'u');
+	config.flags.unicodeSet = hasFlag(flags, 'v');
```

Comment:
Let’s continue matching the `RegExp.prototype` getter name, which in this case is [`unicodeSets`](https://arai-a.github.io/ecma262-compare/?pr=2418#sec-get-regexp.prototype.unicodesets).